app/models/user.rb

```diff
@@ -11,4 +11,18 @@ def self.from_omniauth(auth_info)
     user.save
     user
   end
+
+  def logged_mood?
+    entry = UserMood.where(user_id: id,
+                           created_at: Time.zone.now.beginning_of_day..Time.zone.now.end_of_day)
```

Comment:
This was a clever way to determine if the user logged their mood that day. Nice!

Reply:
Should the User have many Moods through UserMoods?